src/tokenizer.js

```diff
@@ -35,28 +34,37 @@ ace.define('ace/ext/antlr4/tokenizer', ['antlr4/index'], function (require, expo
    *
    * @constructor
    */
-  var Antlr4Tokenizer = function (Lexer, antlrTokenNameToAceTokenType) {
+  var Antlr4Tokenizer = function (Lexer, antlrTokenNameToAceTokenTypeOutside) {
     this.Lexer = Lexer;
-    this.antlrTokenNameToAceTokenType = antlrTokenNameToAceTokenType || {};
+    console.log("Is antlrTokenNameToAceTokenType present?", antlrTokenNameToAceTokenTypeOutside);
+    this.antlrTokenNameToAceTokenType = {};
+    if(antlrTokenNameToAceTokenTypeOutside){
+      this.antlrTokenNameToAceTokenType = antlrTokenNameToAceTokenTypeOutside
+    }
   };
 
   (function () {
     this.getLineTokens = function getLineTokens(line) {
+      console.group("Tokenizer getting tokens");
+      console.log("For the line ", line);
```

Comment:
Change these to console.debug

Reply:
Apparently these will be removed later and another component has a reimplementation of these functions, it is fine to leave them as is